src/course-outline/configure-modal/AdvancedTab.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import { Form } from '@edx/paragon';
+import { useIntl, FormattedMessage, injectIntl } from '@edx/frontend-platform/i18n';
+import { DatepickerControl, DATEPICKER_TYPES } from '../../generic/datepicker-control';
+import messages from './messages';
+import { convertToStringFromDate } from '../../utils';
+
+const AdvancedTab = ({
+  isTimeLimited,
+  setIsTimeLimited,
+  defaultTimeLimit,
+  setDefaultTimeLimit,
+}) => {
+  const intl = useIntl();
+
+  const formatHour = (hour) => {
+    const hh = Math.floor(hour / 60);
+    const mm = hour % 60;
+    let hhs = `${hh}`;
+    let mms = `${mm}`;
+    if (hh < 10) {
+      hhs = `0${hh}`;
+    }
+    if (mm < 10) {
+      mms = `0${mm}`;
+    }
+    if (Number.isNaN(hh)) {
+      hhs = '00';
+    }
+    if (Number.isNaN(mm)) {
+      mms = '00';
+    }
+    return `${hhs}:${mms}`;
+  };
+
+  const [timeLimit, setTimeLimit] = useState(convertToStringFromDate(moment(formatHour(defaultTimeLimit), 'HH:mm')));
+
+  const handleChange = (e) => {
+    if (e.target.value === 'timed') {
+      setIsTimeLimited(true);
+    } else {
+      setDefaultTimeLimit(null);
+      setIsTimeLimited(false);
+    }
+  };
+
+  const setCurrentTimeLimit = (dateStr) => {
+    if (dateStr) {
+      const minutes = moment.duration(moment(dateStr).format('HH:mm')).asMinutes();
+      setDefaultTimeLimit(minutes);
+      setTimeLimit(dateStr);
+    }
+  };
+
+  return (
+    <>
+      <h5 className="mt-4 text-gray-700"><FormattedMessage {...messages.setSpecialExam} /></h5>
+      <hr />
+      <Form.RadioSet
+        name="specialExam"
+        onChange={handleChange}
+        value={isTimeLimited ? 'timed' : 'none'}
+      >
+        <Form.Radio value="none">
+          <FormattedMessage {...messages.none} />
+        </Form.Radio>
+        <Form.Radio value="timed">
+          <FormattedMessage {...messages.timed} />
+        </Form.Radio>
+        <Form.Text><FormattedMessage {...messages.timedDescription} /></Form.Text>
+      </Form.RadioSet>
+      { isTimeLimited && (
+        <div className="hide-header mt-3" data-testid="advanced-tab-hours-picker-wrapper">
+          <DatepickerControl
+            type={DATEPICKER_TYPES.time}
+            value={timeLimit}
```

Comment:
Based on the helper text, the time limit should be able to enter a time greater than 23:59 such as 72:00

Reply:
I would just update it use Form control because the autosuggest  is not necessary and crowds the modal. If we are worried about users not using the correct format for the time, we could add "HH:MM" as the placeholder for the field or set a pattern for the field.

Reply:
@KristinAoki That makes the code simpler. Added a form control with pattern validation as well as placeholder.

![image](https://github.com/openedx/frontend-app-course-authoring/assets/10894099/6758fd29-0b5d-4e4b-9487-0690cd789983)